### PR TITLE
Added support to write to multiple topics from a single producer via Producer#write_to_topic

### LINKED ISSUE #13: Feature request: pls allow for single-connection apps / pushes to multiple topics
I'm coding an app now, and suddenly realizing that using `nsq-ruby`, I need to create multiple connections to the same nsqd for _each_ topic that I want to push to. Looking at the official go client (https://godoc.org/github.com/nsqio/go-nsq#Publish), I'm concluding that this isn't a requirement (that 1 connection can only be dedicated to 1 topic), so is it possible to make the following possible?

1. remove necessity to specify topic for `Nsq::Producer.new`
2. make it possible to specify a topic when doing a `write`

Thanks.

### PR DESCRIPTION
**Pull request by bschwartz**
@jf wanted to write to multiple topics from a single producer. It's a small change and makes sense to me. Decided to go with the most explicit option: adding a new method, `write_to_topic`. This makes it so we won't break `write` functionality.

@jf, will this do the trick?
@MaxPower15 @freerobby @ryanartecona Any thoughts?

Fixes #13

### CODE REVIEW

**Comment by ryanartecona** [comment]
:100:

File: README.md [hunk 1]
```diff
@@ -107,6 +107,22 @@ connection to nsqd fails, you can lose messages. This is acceptable for our use
 cases, mostly because we are sending messages to a local nsqd instance and
 failure is very rare.
 
+
+### `#write_to_topic`
+
+Publishes one or more messages to nsqd. Like `#write`, but allows you to specify
+the topic. Use this method if you want a single producer instance to write to
```
Comment (jf) [inline README.md]: Excellent. Love this note here! :+1:

**Comment by jf** [comment]
Love the excellent note about the usage (`Use this method if you want a single producer instance to write to multiple topics.`). Always love little explanatory notes like this!

I see that `write_to_topic` doesnt call `to_s` on the given topic, though, so it looks like you can't give it a symbol. Would be nice to be able to do so!

File: lib/nsq/producer.rb [hunk 2]
```diff
@@ -30,6 +30,15 @@ def initialize(opts = {})
 
 
     def write(*raw_messages)
+      if !@topic
+        raise 'No topic specified. Either specify a topic when instantiating the Producer or use write_to_topic.'
+      end
+
+      write_to_topic(@topic, *raw_messages)
```
Comment (jf) [inline lib/nsq/producer.rb]: :+1:

**Comment by bschwartz** [comment]
@jf Just added a spec for passing a symbol as topic. Works great.

**Comment by jf** [comment]
Thanks, @bschwartz ! I look forward to switching to a single connection now....

**Pull request merged**